src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml

```diff
@@ -91,8 +91,8 @@
    <style:text-properties fo:hyphenate="false"/>
   </style:style>
   <style:style style:name="Normal" style:family="paragraph">
-   <style:paragraph-properties fo:margin-top="0.141cm" fo:margin-bottom="0.141cm" fo:hyphenation-ladder-count="no-limit"/>
-   <style:text-properties style:font-name="Liberation Sans" fo:font-family="&apos;Liberation Sans&apos;" style:font-style-name="Regular" style:font-family-generic="swiss" style:font-pitch="variable" style:font-size-complex="12pt" fo:hyphenate="false"/>
+   <style:paragraph-properties fo:line-height="normal" fo:margin-top="0.141cm" fo:margin-bottom="0.141cm" fo:hyphenation-ladder-count="no-limit"/>
+    <style:text-properties style:use-window-font-color="false" style:font-name="Liberation Sans" fo:font-family="&apos;Liberation Sans&apos;" style:font-style-name="Regular" style:font-family-generic="swiss" style:font-pitch="variable" style:font-size-complex="12pt" fo:hyphenate="false" />
```

Comment:
style:use-window-font-color="false" is ignored by LO import filter, so not sure what the point is; if you want to specify a particular color use fo:color attribute

Reply:
Good to know, my intention was to overwrite the implicit parent default style. We need to reorder/redesign template styles in ODF 1.4. Would keep it for now.